nix/tools/bazel-cc-toolchain/default.nix

```diff
@@ -27,10 +27,10 @@ let cc-darwin =
       ln -sf $i $out/bin
     done
 
-    # Override clang
-    rm $out/bin/clang
+    # Override cc
+    rm $out/bin/cc
```

Comment:
Is `clang` a symlink to `cc` or the other way around? If `cc` is a symlink to `clang` this seems worse. If `clang` is a symlink to `cc` this seems sensible. If there is no relationship should we patch both? This patch clearly did something before so I’m slightly scared that we now end up not picking up this patch in some cases.

Reply:
`cc` is a symlink to `clang` in nixpkgs's `clang-wrapper`. For the purposes of this toolchain, `nixpkgs_configure_cc_hermetic` only looks at `cc` and `clang` is ignored. Nixpkgs provides these `cc` links consistently across platforms, so it seems more stable to use those.

Though having `clang` around as well might lead to confusion. Maybe it's best to just remove `clang(++)` from this derivation.